src/components/Tutorial/Tutorial2StepContext2.native.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment } from 'react'
+
+import { TUTORIAL_CONTEXT, TUTORIAL_CONTEXT2_PARENT } from '../../constants'
+
+import TutorialHint from './TutorialHint'
+import { Text } from '../Text.native'
+import { headValue } from '../../util'
+import { commonStyles } from '../../style/commonStyles'
+import { Child, Path } from '../../@types'
+
+type TutorialChoice = typeof TUTORIAL_CONTEXT
+
+interface IComponentProps {
+  cursor: Path
+  rootChildren: Child[]
+  tutorialChoice: keyof TutorialChoice
+}
+
+const { smallText, italic } = commonStyles
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const Tutorial2StepContext2 = ({ tutorialChoice, rootChildren, cursor }: IComponentProps) => (
+  <Fragment>
+    <Text style={smallText}>
+      Now add a thought with the text "{TUTORIAL_CONTEXT[tutorialChoice]}"{' '}
+      <Text style={[smallText, italic]}>within</Text> “{TUTORIAL_CONTEXT2_PARENT[tutorialChoice]}”.
+    </Text>
+    {
```

Comment:
all these ternarys are real ugly. would be nice to split the logic outside of the jsx